src/google-authenticator.c

```diff
@@ -45,7 +45,7 @@
 #define BYTES_PER_SCRATCHCODE     4           // 32bit of randomness is enough
 #define BITS_PER_BASE32_CHAR      5           // Base32 expands space by 8/5
 
-static enum { QR_UNSET=0, QR_NONE, QR_ANSI, QR_UTF8 } qr_mode = QR_UNSET;
+static enum { QR_UNSET=0, QR_NONE, QR_ANSI, QR_ANSIi, QR_ANSIg, QR_UTF8, QR_UTF8i, QR_UTF8g } qr_mode = QR_UNSET;
```

Comment:
Please retain existing all-caps stile for these. maybe with `_GREY` and `_INVERSE` suffix.

This also goes for the cmdline option, since without already knowing what the possibilities are, a user will basically not be able to guess.